packages/datadog-instrumentations/src/couchbase.js

```diff
@@ -0,0 +1,155 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource,
+  bindEmit
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+addHook({ name: 'couchbase', file: 'lib/bucket.js', versions: ['^2.6.5'] }, Bucket => {
+  const startChn1qlReq = channel('apm:couchbase:_n1qlReq:start')
```

Comment:
Channels are for operation, not functions, so the names should reflect the operation and not the function name.